Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [MeetingBar/App](https://hosted.weblate.org/projects/meetingbar/app/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/meetingbar/app/horizontal-auto.svg)

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Added localized “Links” tab, status‑bar ongoing‑meeting visibility/timing options, a day‑timeline visual toggle, and a notification “Dismiss” action.

* **Bug Fixes / Improvements**
  * Revised UI copy across Portuguese (pt‑BR), Spanish, German, Croatian, Polish and Dutch for clearer labels, prompts and notifications; adjusted status‑bar icon label wording.

* **Other**
  * Added a Simplified Chinese localization placeholder (empty file).
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Lucas-FcNw committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 35.1% (98 of 279 strings)

Translation: MeetingBar/App
Translate-URL: https://hosted.weblate.org/projects/meetingbar/app/pt_BR/

douugdev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 35.1% (98 of 279 strings)

Translation: MeetingBar/App
Translate-URL: https://hosted.weblate.org/projects/meetingbar/app/pt_BR/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 53.7% (150 of 279 strings)

Translation: MeetingBar/App
Translate-URL: https://hosted.weblate.org/projects/meetingbar/app/pt_BR/

elliecolorado committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (279 of 279 strings)

Translation: MeetingBar/App
Translate-URL: https://hosted.weblate.org/projects/meetingbar/app/es/

GalaxyKeks21 committed:
Translated using Weblate (German)

Currently translated at 96.0% (268 of 279 strings)

Translation: MeetingBar/App
Translate-URL: https://hosted.weblate.org/projects/meetingbar/app/de/

ckuehlbach committed:
Translated using Weblate (German)

Currently translated at 100.0% (279 of 279 strings)

Translation: MeetingBar/App
Translate-URL: https://hosted.weblate.org/projects/meetingbar/app/de/

milotype committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (279 of 279 strings)

Translation: MeetingBar/App
Translate-URL: https://hosted.weblate.org/projects/meetingbar/app/hr/

weblate committed:
Merge remote-tracking branch 'origin/master'

mskopp reviewed (approved):
I approve the german translations

7Luminaries committed:
Added translation using Weblate (Chinese (Simplified Han script))

Commit:
Translated using Weblate (Polish)

Currently translated at 98.5% (275 of 279 strings)

Translation: MeetingBar/App
Translate-URL: https://hosted.weblate.org/projects/meetingbar/app/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 99.2% (277 of 279 strings)

Translation: MeetingBar/App
Translate-URL: https://hosted.weblate.org/projects/meetingbar/app/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 100.0% (279 of 279 strings)

Translation: MeetingBar/App
Translate-URL: https://hosted.weblate.org/projects/meetingbar/app/pl/

matthiasdebaat committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (279 of 279 strings)

Translation: MeetingBar/App
Translate-URL: https://hosted.weblate.org/projects/meetingbar/app/nl/

flytothehighest committed:
Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 3.5% (10 of 279 strings)

Translation: MeetingBar/App
Translate-URL: https://hosted.weblate.org/projects/meetingbar/app/zh_Hans/
